homeassistant/components/samsungtv/media_player.py

```diff
@@ -59,6 +59,9 @@
 # Max delay waiting for app_list to return, as some TVs simply ignore the request
 APP_LIST_DELAY = 3
 
+# Coordinator is used to centralize the data updates
+PARALLEL_UPDATES = 0
```

Comment:
But can you execute multiple action calls at the same time?

Reply:
Yup, nothing bad happens.